chapter2/tests/mkcert/nginx.conf

```diff
@@ -0,0 +1,87 @@
+user nginx nginx;
+worker_processes 1;
+pid /run/nginx.pid;
+include /etc/nginx/modules-enabled/*.conf;
+
+events {
+	worker_connections 768;
+	# multi_accept on;
+}
+
+http {
+
+	##
+	# Basic Settings
+	##
```

Comment:
such comment style make reading much harder. Your comments should provide some values, but right now they're too basic to provide any value.